src/widgets/mentors-register/ui/mentors-register.module.scss

```diff
@@ -0,0 +1,37 @@
+.mentoring-register {
+  display: flex;
+  flex-direction: column;
+
+  @include media-laptop-medium {
+    flex-direction: column;
+    align-items: flex-start;
+
+    * {
+      max-width: 80%;
+    }
+  }
+
+  @include media-tablet {
+    * {
+      max-width: 100%;
+    }
+  }
```

Comment:
i think this `max-width` is not worth it. If you wanted to make title to look prettier, you could've just add  `text-wrap: balance;`

Reply:
Done